data/FloraLucid_2020/metadata.yml

```diff
@@ -0,0 +1,331 @@
+source:
+  primary:
+    key: FloraLucid_2020
```

Comment:
We'd better rename this. Lucid is not the name of a flora but a platform hosting many id keys. See https://www.lucidcentral.org

So maybe `FloraATRP`?

That said, these names seems inconsistent with other AusTraits entries, e.b. ANBG_2019, SAH_2019, etc

Reply:
It is now `ATRP_2020`